### DISCUSSION TITLE
[Client SDK Feature Request] - Add a function to check if Mina public key is valid

### DISCUSSION THREAD

**Discussion by MartinMinkov** [MinaProtocol/mina#9763] [Ideas]
**Description**

Add a method to the Client SDK to verify if a public key is a valid public key.

**Usage**
```
const bIsValid = MinaSDK.verifyPublicKey("B62..."); 
```

This method would be helpful for applications that accept Mina addresses as input and want to do some sort of validation before continuing.

**Comment by MartinMinkov** [comment]
cc @jasongitmail

**Comment by jasongitmail** [comment]
Thanks for creating this thread. Or `MinaSDK.verifyAddress()`?

**Comment by MartinMinkov** [comment]
Note: The Client SDK exports a function called `publicKeyToRaw` which is used for Rosetta. This method does check the validity of a public key, so this _could_ be used to check instead.
